Lua/Comments.tmPreferences

```diff
@@ -15,6 +15,18 @@
 				<key>value</key>
 				<string>-- </string>
 			</dict>
+			<dict>
+				<key>name</key>
+				<string>TM_COMMENT_START_2</string>
+				<key>value</key>
+				<string>--[[</string>
+			</dict>
+			<dict>
+				<key>name</key>
+				<string>TM_COMMENT_END_2</string>
+				<key>value</key>
+				<string>--]]</string>
```

Comment:
After some quick testing, Sublime's lexer is able to comment and uncomment when a multiline string is within the commented area with `--[[--]]`, but not with `--[[]]`.